src/cicd/pipelines.ts

```diff
@@ -195,7 +206,11 @@ export class CICDPipelineStack extends cdk.Stack {
     if (this.pipeline === undefined) {
       throw new Error("`.buildPipeline()` needs to be called first before adding application stages to the pipeline.");
     }
-    const manualApprovals = props.manualApprovals ?? false; // || this._config.get_env_config(stage_id).get('manual_approvals');
+    const manualApprovals = props.manualApprovals
+      ? props.manualApprovals
+      : this.environmentId
+      ? this.config.getEnvConfig("manual_approvals")
+      : false;
```

Comment:
Since these conditions are just checking for `null`, would a null coalescing operator work the same way?
```javascript
const manualApprovals = props.manualApprovals ?? this.config("manual_approvals") ?? false;
```

If not, I wonder if you could indent this in a way that made it clearer that `: this.environmentId` is just a nested condition. Otherwise this chaning of ternary operators is quite hard to read.

Reply:
Yea this is alot cleaner, and works as expected. We're just trying to set based on precedence.